R/layouts.R

```diff
@@ -237,8 +237,7 @@ layout_glue_colors <- layout_glue_generator(
 #' log_info(42)
 #' log_info("ok {1:3} + {1:3} = {2*(1:3)}")
 #' }
-layout_json <- function(fields = c("time", "level", "ns", "ans", "topenv", "fn", "node", "arch",
-                                   "os_name", "os_release", "os_version", "pid", "user", "msg")) {
+layout_json <- function(fields = default_fields()) {
```

Comment:
This was just the first technique that occurred to me to eliminate the lintr warning here. Let me know if there's an approach you'd prefer, or if you'd just like me to think about this more.